pyproject.toml

```diff
@@ -62,6 +63,9 @@ environments = [
     "python_version > '3.9'"
 ]
 
+[tool.uv.sources]
+vllm = { git = "https://github.com/vllm-project/vllm", rev = "v0.11.1" }
```

Comment:
Installing vllm this way (with `VLLM_TARGET_DEVICE=empty`) leaves out extra cuda-only dependencies from the uv.lock, since the published vllm wheels on pypi are only built for cuda.